i18n.ts

```diff
@@ -5,10 +5,33 @@ import * as translations from './translations';
 export interface SupportedLanguage {
   id: string;
   label: string;
+  dateFormat: string;
 }
 
+export type Lang = 'en' | 'es';
+
+export const DEFAULT_LOCALE: Lang = 'es';
```

Comment:
Should we make english the default?  That will arguably be the most used language for the labs.

Reply:
You are correct. My mistake.